src/gyazo.rb

```diff
@@ -1,15 +1,26 @@
 #!/usr/bin/env ruby
 
-# setting
-browser_cmd = 'xdg-open'
-clipboard_cmd = 'xclip'
-
 require 'net/http'
 require 'open3'
 require 'openssl'
 require 'json'
 require 'yaml'
 
+# setting
+configfile = "#{ENV['HOME']}/.gyazo.config.yml"
+if File.exist?(configfile) then
+  config = YAML.load_file(configfile)
+end
+
+browser_cmd = config == nil ? 'xdg-open' : config['browser_cmd']
```

Comment:
if `browser_cmd` is not configured in the yml file, this line will fail.

Plus, could this be more shorter like this?
`browser_cmd = config['browser_cmd'] || 'xdg-open'`